src/FastRouteRouterFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Router;
+
+use Interop\Container\ContainerInterface;
+
+/**
+ * Create and return an instance of FastRouteRouter.
+ *
+ * Configuration should look like the following:
+ *
+ * <code>
+ * 'router' => [
+ *     'fastroute' => [
+ *         'cache_enabled' => true, // true|false
+ *         'cache_file'   => '(/absolute/)path/to/cache/file', // optional
+ *     ],
+ * ]
+ * </code>
+ */
+class FastRouteRouterFactory
```

Comment:
Hi, I believe we should allow users to choose weather to use caching or not. 
Modifying the factory so that it can parse some properly named config keys (the config keys i chose are just an example, it's what I use) and passing them to a modified Router constructor makes it more flexible. Otherwise one would have to use a custom factory to pass caching-related config parameters.